theano/tensor/basic.py

```diff
@@ -1733,6 +1733,7 @@ def infer_shape(self, node, shapes):
         return [rval]
 
     def R_op(self, inputs, eval_points):
+        raise ValueError
```

Comment:
What about this line? I was getting errors that argmax is nondifferentiable yet not raises when I was testing without it.